skycultures/greek_farnese/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+########### install files ###############
+
+# install info.ini
+INSTALL(FILES info.ini DESTINATION ${SDATALOC}/skycultures/greek_farnese )
+
+# install constellations data, native names data, artwork and descriptions
+INSTALL(DIRECTORY ./ DESTINATION ${SDATALOC}/skycultures/greek_farnese
+	FILES_MATCHING PATTERN "constellation*" 
+	PATTERN "*_names.fab"
+	PATTERN "description.*.utf8"
+	PATTERN "*.png"

```

Comment:
You have PNG and JPG images in this SC, so, please add line `PATTERN "*.jpg"` after `PATTERN "*.png"` line